### DISCUSSION TITLE
Can I just use JavaScript?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [microsoft/fast#6663] [Q&A] [answered]
Shopping for a web framework, came across FAST, checked the docs briefly. Main question - can I use FAST without TypeScript, just modern JavaScript andC SS, no transpiling/building/bundling phase.

Also looked at FluentUI - all examples are in TypeScript/Blazor something...

**Comment by chrisdholt** [comment, marked as answer]
This is not exhaustive, but here is an example of a custom element _without_ using TS, experimental decorators, etc: https://stackblitz.com/edit/js-yvkrh3?file=index.js

While stackblitz is likely transpiling/bundling somewhere under the covers, the above example hopefully can illustrate that while FAST provides typings for all packages, we export JavaScript. You should be able to use FAST without TS and any transpiling/bundling. You can also leverage a CDN if you want to avoid leveraging the npm packages.

**Reply by hrstoyanov** [reply]
Thanks, will definitely use CDN. Just wanted to make sure FAST is modern enough for my next project.
